### PR TITLE
suggest labels on uploads page

### PR DESCRIPTION
**Pull request by jamesgorrie**
Suggesting the same "super" labels on the upload page to create a nice complete cycle of a workflow.
upload-label-search

### CODE REVIEW

**Comment by jamesgorrie** [comment]
CC @tsop14.

**Comment by tsop14** [comment]
Is 'observer' not actually a label? :scream: 

obvs 'obs-something' would be better?

**Comment by jamesgorrie** [comment]
@tsop14 They have a lot of top-folders, `obs{art/travel/stuff}` seemed to be the obvs choice.
[Here's the full observer list](https://github.com/guardian/grid/blob/master/media-api/app/controllers/MediaApi.scala#L321-L322) - do you think there should be something else?

File: kahuna/public/js/components/gr-preset-labels/gr-preset-labels.html [hunk 1]
```diff
@@ -28,13 +28,19 @@
...
-            <input type="text"
-                   class="show-no-error"
-                   placeholder="e.g. Observer"
-                   gr:auto-focus
-                   required
-                   ng:model="ctrl.newLabel"
-                   ng:disabled="ctrl.adding" />
+        <gr-datalist class="related-labels__datalist"
+                   gr:search="ctrl.suggestedLabelsSearch(q)">
+          <input type="text"
```
Comment (theefer) [inline kahuna/public/js/components/gr-preset-labels/gr-preset-labels.html]: Two tabs!? you stingy :monkey: :tongue:

File: media-api/app/controllers/MediaApi.scala [hunk 2]
```diff
@@ -50,6 +50,9 @@ object MediaApi extends Controller with ArgoHelpers {
 
   val searchLinkHref = s"$rootUri/images{?$searchParamList}"
 
+  private val getSuggestedLabelsLink =
```
Comment (theefer) [inline media-api/app/controllers/MediaApi.scala]: I'd do away with the `get`. It's just a value.

**Comment by theefer** [comment]
Neat :+1:

File: kahuna/public/js/components/gr-preset-labels/gr-preset-labels.html (hunk 1 above)
Comment (jamesgorrie) [inline kahuna/public/js/components/gr-preset-labels/gr-preset-labels.html]: Every space counts

**Pull request merged**